src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -130,6 +130,8 @@
 
     private int minimumNumberOfInstances;
 
+    private MinimumNumberOfInstancesTimeRangeConfig minimumNumberOfInstancesTimeRangeConfig;
```

Comment:
What about old configs not having this property? Should you manage that in readResolve?

Reply:
IIUC There is a nullcheck for its usage

Reply:
Right!

Reply:
Correct, there's a null check. And I did a quick check now and couldn't find anywhere that I had missed checking. :)